theme/views.py

```diff
@@ -4,6 +4,7 @@
 from django.views.generic import TemplateView
 from django.http import HttpResponse
 from django.shortcuts import redirect
+from django.conf import settings
```

Comment:
Does `mezzanine.conf.settings` include all the settings from `settings.py`? If so, I can just remove the django import and use that.